src/esip-2022-presentation/script/benchmark_queries.py

```diff
@@ -143,21 +144,29 @@ def get_json(uri):
 parq_results_uri = ''
 
 
-# Run this block to configure a small set of experiments for testing purposes.
-# This is useful since the Parquet experiments run so slowly currently.
-nb_repeats = 1
-query_map = {'mean_day': query_map['mean_day']}
+nb_repeats = 5
+## query_map = {'mean_day': query_map['mean_day']}
 time_ranges = [
     slice('1990-01-01', '1990-02-01'),
+    slice('1990-01-01', '1991-01-01'),
+    slice('1990-01-01', '2000-01-01'),
 ]
+nb_reaches = len(avail_comids)
+nb_workers = len(client.scheduler_info()['workers'])
```

Comment:
Why does this cell of code needs to be here at all?

Reply:
Removed it!